.eslintrc

```diff
@@ -1,4 +1,6 @@
 {
+    "extends": ["eslint:recommended"],
+    "parser": "@babel/eslint-parser",
```

Comment:
Do we need the custom parser here or can we go with the default? If it is a missing js feature, you might get away with just changing ecmaVersion to catch up.

Reply:
I just checked. If I set `es6` in `env` to `true`, then I do not need the parser from babel.
Is `es6` fine for you? If yes, I'll provide a patch to remove the babel parser and set the corresponding eslint option.

Reply:
You should be able to get by by adding `"es2024": true` to the `env` section, no parser at all.

Reply:
Yes, exactly. Is there any specific ES version that you would like to target in the project? If not, I will just use the lowest one that allows `eslint` to work, i.e., `es6: true`

Reply:
Oh duh, sorry, I misread your comment, for some reason I was thinking you'd be using a parser named es6. Go ahead with es6: true in env!

Reply:
Okay, I have added the changes, i.e., added `es6: true` and removed all traces of `@babel/eslint-parser`